doc/ldoc/assets/ldoc.ltp

```diff
@@ -0,0 +1,420 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
```

Comment:
Do we really need to push this generated code? I guess CI should push it github.io or at least we should have protection when we change something and don't update a doc.

Reply:
it's a template used by ldoc:

```
sergeyb@pony:~/sources/MRG/expirationd$ make apidoc 
.rocks/bin/ldoc -c /home/sergeyb/sources/MRG/expirationd/doc/ldoc/config.ld \
             -d /home/sergeyb/sources/MRG/expirationd/doc/apidoc/ expirationd.lua
reading configuration from /home/sergeyb/sources/MRG/expirationd/doc/ldoc/config.ld
template not found at 'doc/ldoc/assets' Use -l to specify directory containing ldoc.ltp
make: *** [Makefile:19: apidoc] Error 255
```